WalletWasabi.Fluent/ViewModels/CoinControl/SelectCoinsDialogViewModel.cs

```diff
@@ -28,7 +28,7 @@ public partial class SelectCoinsDialogViewModel : DialogViewModelBase<IEnumerabl
 {
 	public SelectCoinsDialogViewModel(WalletViewModel walletViewModel)
 	{
-		var pockets = walletViewModel.Wallet.GetPockets();
+		var pockets = walletViewModel.Wallet.GetPockets(onlyAvailableCoins: true);
```

Comment:
> And btw it will be handled by disabling the selection for the coins that are CJing.

I don't get this. Where is this filtering? Should I implement it or is it already on master?